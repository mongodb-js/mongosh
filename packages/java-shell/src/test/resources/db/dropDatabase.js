// command
use dbToDrop
// command
db.dropDatabase();
