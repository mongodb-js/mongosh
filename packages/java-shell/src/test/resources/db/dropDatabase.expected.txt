switched to db dbToDrop
{ "ok": 1, "dropped": "dbToDrop" }